src/libs/parsers/formatDistance.ts

```diff
@@ -14,7 +14,7 @@ export const getHumanizeRelativeDistance = (
   if (!userLat || !userLng || !venueLat || !venueLng) return
 
   const distanceInMeters = computeDistanceInMeters(venueLat, venueLng, userLat, userLng)
-  return humanizeDistance(distanceInMeters)
```

Comment:
Je vois le commentaire "Most functions are taken from the code of the browser" : ce genre d'implémentation custom me fait un peu peur :/ .
Il n'y a pas moyen d'utiliser une librairie tierce (populaire et maintenue de manière pro) ou une API donnée par React Native (e.g. `Intl.NumberFormat`) ?
@Giregm tâche tech à prévoir pour le bien long terme du projet ?

Reply:
moi ça me dérange pas perso. ça répond à un besoin bien précis, c'est clair et bien testé

Reply:
Je suis d'accord qu'on a repris ses fonctions mais pas sans les regarder. 
Elles sont en effet bien testées et respectent probablement comme il se doit les règles de gestions du projet. 

Probablement qu'on ne fera pas plus fiable qu'une librairie tierce mais en utiliser une ne fitera jamais notre besoin précis, j'imagine qu'on aura toujours une couche d'abstraction spécifique au projet à ajouter derrière.

Je suis pour enlever le commentaire `Most functions are taken from the code of the browser` et assumer de nous approprier ces fonctions.

Qu'en pensez vous ?

Reply:
Je trouve dommage de pas utiliser des librairies d'internationalization, car il existe des standards par langue et c'est dommage de ne pas les utiliser. A priori utiliser des virgules est un le standard de base pour les nombres écris en français. 
Vous repenserez à ce commentaire le jour où on devra supporter d'autres langues :).
Ok pour enlever le commentaire sur le  ode repris de la webapp.